packages/headless/src/controllers/commerce/core/breadcrumb-manager/headless-core-breadcrumb-manager.ts

```diff
@@ -236,7 +236,10 @@ export function buildCoreBreadcrumbManager(
           .map((facetId) =>
             options.facetResponseSelector(engine[stateKey], facetId)
           )
-          .filter((facet): facet is AnyFacetResponse => facet !== undefined)
+          .filter(
+            (facet): facet is AnyFacetResponse =>
+              facet !== undefined && facet.values.length > 0
```

Comment:
Looks good, only question I'd have is does it mirror the behavior of non-commerce breadcrumb manager controllers?

Reply:
Yup, the non-commerce one excludes breadcrumbs with no values: https://github.com/coveo/ui-kit/blob/3330e49f180b7b997861446cf23bad2ea28cab51/packages/headless/src/controllers/core/breadcrumb-manager/headless-core-breadcrumb-manager.ts#L249